client/app/email-domain/order/email-domain-order.html

```diff
@@ -19,18 +19,32 @@ <h1 data-i18n-static="mxPlan_order_title"></h1>
         <!-- step 1 -->
         <h2 data-i18n-static="mxPlan_order_step1_title"></h2>
 
-        <div class="form-group">
-            <label class="oui-label" for="select-order-domain"
-                   data-i18n-static="mxPlan_order_step1_domain"></label>
-            <div class="oui-select oui-select_inline">
-                <select class="oui-select__input" id="select-order-domain" name="orderDomain"
-                        data-ng-change="ctrlMXPlanOrder.fetchOffers()"
-                        data-ng-model="ctrlMXPlanOrder.selectedOrder.config.domain"
-                        data-ng-options="domain for domain in (ctrlMXPlanOrder.list.domains | orderBy) track by domain">
-                    <option value="" disabled
-                            data-i18n-static="select_option"></option>
-                </select>
-                <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
+        <div class="row">
+            <div class="form-group col-md-4">
+                <label class="oui-label" for="select-order-domain"
+                    data-i18n-static="mxPlan_order_step1_domain"></label>
+                <div class="oui-select oui-select_inline">
+                    <select class="oui-select__input" id="select-order-domain" name="orderDomain"
+                            data-ng-change="ctrlMXPlanOrder.fetchOffers()"
+                            data-ng-model="ctrlMXPlanOrder.selectedOrder.config.domain"
+                            data-ng-options="domain for domain in (ctrlMXPlanOrder.list.domains | orderBy) track by domain">
+                        <option value="" disabled
+                                data-i18n-static="select_option"></option>
+                    </select>
+                    <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
+                </div>
+            </div>
+            <div class="col-md-6 col-md-offset-1">
+                <div class="oui-message oui-message_warning" role="alert">
+                    <i class="oui-icon oui-icon-warning_circle oui-icon_bicolor" aria-hidden="true"></i>
```

Comment:
`<span>`

Reply:
Doesn't work fine with some `oui-icon`, this is why I chose the `i` tag

Reply:
It actually works with `span` so I corrected